src/lib/components/QuestionCard.svelte

```
@@ -23,35 +23,38 @@
+		<Card.Description class="text-base/normal font-medium"
+			>{question.question_text}{#if question.is_mandatory}<span class="ml-1 text-red-500">*</span>
+			{/if}</Card.Description
+		>
+	</Card.Header>
+
+	<Card.Content class="p-5 pt-1">
+		<RadioGroup.Root
+			onValueChange={(value) => {
+				handleSelection(question.id, value);
+			}}
+			value={selectedQuestions.find((item: any) => item.question === question.id)?.response}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)